pennylane/ops/qubit/observables.py

```diff
@@ -244,12 +244,22 @@ def __init__(self, H, wires=None, id=None):
         if not isinstance(H, csr_matrix):
             raise TypeError("Observable must be a scipy sparse csr_matrix.")
         super().__init__(H, wires=wires, id=id)
+        self.H = H
         mat_len = 2 ** len(self.wires)
         if H.shape != (mat_len, mat_len):
             raise ValueError(
                 f"Sparse Matrix must be of shape ({mat_len}, {mat_len}). Got {H.shape}."
             )
 
+    def __mul__(self, value):
+        r"""The scalar multiplication operation between a scalar and a SparseHamiltonian."""
+        if not isinstance(value, (int, float)):
```

Comment:
this is not a major concern because scipy data is already non-differentiable, but perhaps we should allow for scalars that are not vanilla python types, eg. `np.array(1.5)`? I have confirmed locally that you can pass numpy arrays to `csr_matrix.multiply`. this could probably be tested simply with `if qml.math.ndim(value) != 0`

Reply:
Gr8 point — will add!